core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -280,6 +280,8 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
 
   private[spark] def ui: Option[SparkUI] = _ui
 
+  def uiWebUrl: Option[String] = _ui.map(_.webUrl)
```

Comment:
`def uiWebUrl: Option[String] ` method is called inside repl's `processLine` so having `private[spark]` won't be enough. It returns a "cannot be accessed" error.

Alternatively, I can change the `def ui: Option[SparkUI]` method from `private[spark]` to `public` and then change the print statement to 
`_sc.ui.foreach(ui => println(s"Spark context Web UI available at ${ui.webUrl}"))`

That would work as well but I preferred to have a new public method (`uiWebUrl `) rather than making the `ui` public. I would be happy to hear your suggestion.